Metro-Records/la-metro-dashboard
# Update production environment's scraper image tag config

## Overview

I noticed that the production environment file was set to pull the `main` scraper image instead of `deploy`. This PR updates the environment file.

## Testing Instructions

 * Decrypt `configs/production/.env` with `blackbox_postdeploy`
 * Verify that `LA_SCRAPERS_DOCKER_IMAGE_TAG` has been updated to `deploy`

Connects Metro-Records/scrapers-lametro#10
